src/univariate/discrete/negativebinomial.jl

```diff
@@ -132,6 +132,10 @@ function laplace_transform(d::NegativeBinomial, t)
 end
 
 mgf(d::NegativeBinomial, t::Real) = laplace_transform(d, -t)
+function cgf(d::NegativeBinomial, t)
+    r,p = params(d)
+    r*cgf(Geometric(p),t)
```

Comment:
```suggestion
    r * cgf(Geometric{typeof(p)}(p), t)
```

Reply:
Thanks a lot I was not aware of this trick!